test/functional/test_framework/authproxy.py

```diff
@@ -112,6 +112,16 @@ def _request(self, method, path, postdata):
                 return self._get_response()
             else:
                 raise
+        except OSError as e:
+            if e.errno == 42:
```

Comment:
The errno is actually 41; to avoid hardcoding the value, I would use `from errno import EPROTOTYPE` and then:
```
if e.errno == EPROTOTYPE:
   ...
```

Reply:
Oops, good catch.